src/Bijectors.jl

```diff
@@ -135,6 +135,11 @@ invlink(d::Distribution, y) = inverse(bijector(d))(y)
 
 # To still allow `logpdf_with_trans` to work with "batches" in a similar way
 # as `logpdf` can.
+
+# Default catch-all so we can work with distributions by default and batch-support can be
+# added when needed.
+_logabsdetjac_dist(d, x) = logabsdetjac(bijector(d), x)
```

Comment:
It seems (based on https://github.com/TuringLang/Bijectors.jl/blob/7c8b533e4563f5a89b53dddc069e029e655ee370/src/Bijectors.jl#L167) we could be a bit more restrictive:
```suggestion
_logabsdetjac_dist(d::Distribution, x) = logabsdetjac(bijector(d), x)
```